buildbot/osuosl/master/config/builders.py

```diff
@@ -2792,4 +2792,45 @@
                     env={
                         'CCACHE_DIR' : WithProperties("%(builddir)s/ccache-db"),
                     })},
+
+    # Builders similar to used in Buildkite premerge pipeline.
+    # Please keep in sync with llvm-project/.ci configurations.
+
+    # See https://github.com/llvm/llvm-project/blob/main/.ci/monolithic-windows.sh.
+    {'name' : "premerge-monolithic-windows",
+    'tags'  : ["premerge"],
+    'collapseRequests': True,
```

Comment:
If you want to collapse build requests, just skip the `collapseRequests` arg and let the default machinery do the job. Setting it to True is a bad idea, as there is a number of build requests which should not be collapsed, like manual build requests, for example, or requested clean builds, and such.

Reply:
Done.